src/test/kotlin/org/elm/ide/lineMarkers/ElmRecursiveCallLineMarkerProviderTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.elm.ide.lineMarkers
+
+class ElmRecursiveCallLineMarkerProviderTest : ElmLineMarkerProviderTestBase() {
+    fun `test top level functions`() = doTestByText(
+            """
+f a = f (a - 1) --> Recursive call
+g a =
+  g (a + 1) --> Recursive call
+h a = h ( h (a + 1) ) --> Recursive call
+""")
+
+    fun `test nested functions`() = doTestByText(
```

Comment:
I'm guessing that mutual recursion is not detected? It looks like the Kotlin plugin doesn't even do that.

Reply:
No, it's not. I'm not sure of any way to do that efficiently.